libstdc++-v3/include/std/contracts

```diff
@@ -64,32 +98,66 @@ namespace contracts
   };
 
   enum class detection_mode  {
+  // From P3290
+    unspecified = 0,
     predicate_false = 1,
-    evaluation_exception = 2
+    evaluation_exception = 3
```

Comment:
Why 3 instead of 2?

Reply:
typo, it's what was causing the issue you were seeing, fixed